pinot-broker/src/main/java/org/apache/pinot/broker/routing/RoutingManager.java

```diff
@@ -51,7 +51,7 @@
 import org.apache.pinot.common.metrics.BrokerMetrics;
 import org.apache.pinot.common.request.BrokerRequest;
 import org.apache.pinot.common.utils.CommonConstants;
-import org.apache.pinot.common.utils.CommonConstants.Helix.StateModel.RealtimeSegmentOnlineOfflineStateModel;
+import org.apache.pinot.common.utils.CommonConstants.Helix.StateModel.SegmentOnlineOfflineStateModel;
```

Comment:
can we just call it `SegmentStateModel`?

Reply:
@mcvsubbu If we want to further clean it up, we can merge the BrokerStateModel and SegmentStateModel and put all instance states (`ONLINE`, `CONSUMING`, `OFFLINE`, `ERROR`) under `CommonConstants.Helix.InstanceState`. Thoughts?

Reply:
I dont think merging BrokerStateModel and SegmentStateModel is a good idea.  Broker does not have CONSUMING state.

I think the reason these are coded separately (for realtime and offline) is because we did not have CONSUMING state before. By now, the state model in helix should have consuming for all pinot use cases. So, merging these is fine.

Reply:
@mcvsubbu Changed them to `SegmentStateModel` and `BrokerResourceStateModel`.
Also, can you please give me a ship it if it looks good?